aecore_eqc/aec_sync_eqc.erl

```diff
@@ -6,6 +6,9 @@
 %%%      Try to make the software in aehttp to add the local peer, such that we
 %%%      don't need the admin here.
 %%%
+%%%       TODO:
+%%%          - This test is known not to work. It requires at least update from http to noise.
```

Comment:
I attempted to update this test - see https://github.com/Quviq/epoch-eqc/pull/31/commits/f2a1c99ca6c5d112a4ad1d53f82588085edc93f4 - though it is not trivial. So I suggest to at least add a note that test is known not to work (or delete the test...). I tracked updating this test as separate ticket https://www.pivotaltracker.com/story/show/166238158